server/src/middleware/auth.middleware.ts

```diff
@@ -62,6 +62,9 @@ const auth = {
       if (response.success) {
         req.body = { ...req.body, ...response.jwtPayload };
         next();
+        return res
```

Comment:
This will affect all other routes that have this middleware, which is a no-op. Maybe we can add this as a new controller and use it there, instead of modifying the middleware.

Reply:
Done :+1: